### ISSUE TITLE
lineBreakReplacement parameter in FROM_EXCEL table transformer skips white spaces

### ISSUE THREAD

**Issue by yanavera** [vividus-framework/vividus#647]
**Describe the bug**
LineBreakReplacement parameter in FROM_EXCEL table transformer does not work exactly. White spaces are skipped and there is no possibility to change line break to space.

**To Reproduce**
Run attached story file. Excel file used in the story you can find below
[TestStory.txt](https://github.com/vividus-framework/vividus/files/4723889/TestStory.txt)
[test.xlsx](https://github.com/vividus-framework/vividus/files/4723830/test.xlsx)

**Expected behavior**
Multiple line text from excel is extracted as string with spaces between words started with new line

**Artifacts**
 - Allure report
[allure.zip](https://github.com/vividus-framework/vividus/files/4723805/allure.zip)

Execution environment (please complete the following information):
 - Vividus version: 0.2.3-SNAPSHOT
 - OS: Windows 10
 - Java: [13.0.2]

**Additional context**
all possible approaches to specify white space were tested (e.g. \u0020, ,\ ,\u005Cu0020). 
please feel free to contact me for more details
**Priority - p3**

**Comment by valfirst** [comment]
The fix is implemented in JBehave, see https://jbehave.atlassian.net/browse/JBEHAVE-1411.
New JBehave version containing the fix was integrated in #746.

**Comment by valfirst** [comment]
The example was provided in #747:
```
Scenario: Verify FROM_EXCEL transformer
Then `<joined>` is equal to `line 1 line 2 line 3`
Examples:
{transformer=FROM_EXCEL, path=/data/excel.xlsx, sheet=Sheet1, addresses=A1, column=joined, \{lineBreakReplacement|VERBATIM\}= }
```
